### PR TITLE
Added config action to setup variables for a workflow

### PR DESCRIPTION
**Pull request by jcortes**
The idea is to give support to any workflow with variables that can be set with a simple action component step that in a following step we can reference with something like
```
export default defineComponent({
  async run({ steps, $ }) {
    return steps.helper_functions_export_variables.config;
  },
})
```

### CODE REVIEW

**Review by vellames-turing — APPROVED** [review]

**Comment by dannyroosevelt** [comment]
@jcortes is there a reason to `$.export` config in addition to just `return`ing it?

Also, this should go in the `helper_functions` app, not `helpers`.

File: components/helper_functions/actions/export-variables/export-variables.mjs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default {
+  key: "helper_functions-export-variables",
+  name: "Export Variables",
+  description: "Export variables for use in your workflow",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    config: {
```
Comment (dannyroosevelt) [inline components/helper_functions/actions/export-variables/export-variables.mjs:8]: can you also import `helper_functions` so the action inherits the app icon?

File: components/helper_functions/actions/export-variables/export-variables.mjs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export default {
```
Comment (dannyroosevelt) [inline components/helper_functions/actions/export-variables/export-variables.mjs:1]: `import helper_functions from "../../helper_functions.app.mjs"`

File: components/helper_functions/actions/export-variables/export-variables.mjs (hunk 1 above)
Comment (dannyroosevelt) [reply components/helper_functions/actions/export-variables/export-variables.mjs:8]: so that the action inherits the app icon

**Review by dannyroosevelt — CHANGES_REQUESTED** [review]
Can you just make sure to import the app in the action props, so the action inherits the app icon? Then let's ship it!

**Review by dannyroosevelt — APPROVED** [review]

**Pull request merged**